source/reference/mongosync.txt

```diff
@@ -19,11 +19,18 @@
 Description
 -----------
 
-The ``mongosync`` binary is the primary process used in Cluster-to-Cluster Sync.
-It migrates data from one cluster to another and can keep the clusters
-in continuous sync.
+The ``mongosync`` binary is the primary process used in
+{+c2c-product-name+}. ``mongosync`` migrates data from one cluster to
+another and can keep the clusters in continuous sync. 
 
-This document provides a complete overview of all command-line options.
+This document provides a complete overview of the ``mongosync`` command
+line options.
+
+.. note::
+
+   - The cluster must use MongoDB 6.0 or later.
+   - ``mongosync`` supports replica sets and sharded clusters.
+   - Standalone MongoDB instances are not supported.
```

Comment:
Potentially link this https://www.mongodb.com/docs/manual/tutorial/convert-standalone-to-replica-set/ with the word standalone?